terraform/aws/providers.tf

```diff
@@ -0,0 +1,40 @@
+terraform {
+  required_providers {
+    databricks = {
+      source  = "databrickslabs/databricks"
+      version = "~>0.4.0"
+    }
+    aws = {
+      source  = "hashicorp/aws"
+      version = "~>3.68.0"
+    }
+  }
+}
+
+provider "aws" {
+  region  = var.aws_region
+  profile = var.aws_profile
+
+  default_tags {
+    tags = {
+      Vendor = "Databricks Unity Catalog"
+    }
+  }
+}
+
+// initialize provider in "MWS" mode, to add users at account-level
+provider "databricks" {
+  alias      = "mws"
+  host       = "https://accounts.cloud.databricks.com"
+  account_id = var.databricks_account_id
+  username   = var.databricks_account_username
+  password   = var.databricks_account_password
+}
+
+// initialize provider at workspace level, to create UC resources
+// authentication using one of the described approaches at
+// https://registry.terraform.io/providers/databrickslabs/databricks/latest/docs#authentication
+provider "databricks" {
+  alias   = "workspace"
+  profile = "UC-TEST"
```

Comment:
also login to a workspace with the username/password

Reply:
do we want to login to workspace with user/pass or a PAT?

Some customers might not allow user/pass.

Reply:
I'm tempted to just leave it as "choose your own method of auth"

In the step-by-step guide, we can be opinionated and choose user/pass

Reply:
@pohlposition PAT is not necessary in this particular case. the purpose of TF module is to work and the simpler it is, the more chances it has to work :)